modules/apps/digital-signature/digital-signature-web/src/main/resources/META-INF/resources/js/pages/envelope/EnvelopeList.js

```diff
@@ -103,7 +103,7 @@ const EnvelopeList = ({history}) => {
 					emailSubject,
 					name: <Link to={`/envelope/${envelopeId}`}>{name}</Link>,
 					recipients: (
-						<span className="d-flex">
+						<span className="create-date-column">
```

Comment:
@MateusCastro , I don't see any relationship between `.create-date-column` and `recipients` column.

I think you could try to use the classNames directly, because this is a very particular use case.

https://getbootstrap.com/docs/4.4/utilities/flex/


<img width="301" alt="image" src="https://user-images.githubusercontent.com/22279592/120560317-e1962d80-c3d8-11eb-8303-e167128b9f00.png">

Like this example, but just reusing the classes `d-flex flex-wrap`

Reply:
Thanks, Keven. I didn't know there was this class on Bootstrap.I will make the change.

Reply:
Thanks man!!